projects/uma/index.js

```diff
@@ -0,0 +1,38 @@
+const { request, gql } = require("graphql-request");
+const sdk = require('../../sdk')
+
+const endpoint = 'https://api.thegraph.com/subgraphs/name/umaprotocol/mainnet-contracts'
```

Comment:
We prefer to avoid adding any 3rd party api call inside adapter. Please Use our SDK apis to pull on-chain data.

Reply:
Thanks to the team for talking through this with me on Discord! For anyone reading along on GitHub: Since this subgraph query is only getting a list of financial contracts, it was determined that it was fine. The actual value calculation uses the SDK.

Reply:
Actually, re-opening this thread since the GraphQL query does some calculation and may need to be re-written.

Reply:
This is resolved now! No Graph dependencies.